packages/components/listbox/src/option.ts

```diff
@@ -15,7 +15,8 @@ declare global {
  *
  * @slot default - The option's label.
  */
-export class Option extends ScopedElementsMixin(LitElement) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export class Option<T = any> extends ScopedElementsMixin(LitElement) {
```

Comment:
Can we please use `unknown` instead of `any` here?

Reply:
As discussed on Slack, leaving it `any` for now